src/lxqtplatformtheme.cpp

```diff
@@ -149,7 +147,7 @@ void LXQtPlatformTheme::loadSettings() {
     color = highlightedTextColor_;
     highlightedTextColor_.setNamedColor(settings.value(QLatin1String("highlighted_text_color")).toString());
     if (!paletteChanged_)
-        paletteChanged_ = color != highlightedTextColor_.isValid() && color != highlightedTextColor_;
+        paletteChanged_ = color != static_cast<QRgb>(highlightedTextColor_.isValid()) && color != highlightedTextColor_;
```

Comment:
This was my typo, I think. It should be:

```c++
paletteChanged_ = highlightedTextColor_.isValid() && color != highlightedTextColor_;
```

Reply:
Thanks. Fixed.

Reply:
We wouldn't find this mistake of mine if it weren't for your patch.

BTW, you forgot to change it.

Reply:
@tsujan Done in https://github.com/lxqt/lxqt-qtplugin/pull/66.